pkg/validation/test_test.go

```diff
@@ -237,43 +235,45 @@ func TestValidateTests(t *testing.T) {
 					},
 				},
 			},
-			release:       &api.ReleaseTagConfiguration{Name: "origin-v3.11"},
-			expectedValid: true,
+			release: &api.ReleaseTagConfiguration{Name: "origin-v3.11"},
 		},
 		{
 			id: "invalid secret mountPath",
 			tests: []api.TestStepConfiguration{
 				{
-					As:                                       "test",
-					OpenshiftAnsibleClusterTestConfiguration: &api.OpenshiftAnsibleClusterTestConfiguration{},
+					As:                         "test",
+					Commands:                   "commands",
+					ContainerTestConfiguration: &api.ContainerTestConfiguration{From: "src"},
 					Secret: &api.Secret{
 						Name:      "secret",
 						MountPath: "/path/to/secret:exec",
 					},
 				},
 			},
-			expectedValid: false,
+			// TODO: The code actually just checks `filepath.IsAbs()`
```

Comment:
Amazing.  Let's just make the test case value a relative path, I don't think anyone cares.